march_hardware_interface/launch/march4.launch

```diff
@@ -1,7 +1,5 @@
 <launch>
-    <rosparam file="$(find march_hardware_interface)/config/march4/hardware.yaml" command="load"/>
     <rosparam file="$(find march_hardware_interface)/config/march4/controllers.yaml" command="load"/>
```

Comment:
do you need to change controllers everytime you change a joint in the urdf (to fixed or back to revol revolute)?

Reply:
Yes